tests/examples/awful/widget/calendar_popup/cell_properties.lua

```diff
@@ -0,0 +1,40 @@
+--DOC_GEN_IMAGE --DOC_HIDE
```

Comment:
You can use `--DOC_HIDE_START` and `--DOC_HIDE_END` to hide blocks of code at once.

Reply:
Thank you. This is great. I'll try.